### ISSUE TITLE
`Tracking` Visual Studio fixes & improvements

### ISSUE THREAD

**Issue by vzarytovskii** [dotnet/fsharp#14014]
Tracking issue for all VS fixes and improvements:
- https://github.com/dotnet/fsharp/issues/13927
- https://github.com/dotnet/fsharp/issues/13882
- https://github.com/dotnet/fsharp/issues/13951
- https://github.com/dotnet/fsharp/issues/13919
- https://github.com/dotnet/fsharp/issues/13909
- https://github.com/dotnet/fsharp/issues/13714
- https://github.com/dotnet/fsharp/issues/13717
- https://github.com/dotnet/fsharp/issues/13762
- https://github.com/dotnet/fsharp/issues/13482
- https://github.com/dotnet/fsharp/issues/13214
- https://github.com/dotnet/fsharp/issues/12916
- https://github.com/dotnet/fsharp/issues/13133
- https://github.com/dotnet/fsharp/issues/12253
- https://github.com/dotnet/fsharp/issues/12155
- https://github.com/dotnet/fsharp/issues/1784
- https://github.com/dotnet/fsharp/issues/2693
- https://github.com/dotnet/fsharp/issues/3512
- https://github.com/dotnet/fsharp/issues/4116
- https://github.com/dotnet/fsharp/issues/4175
- https://github.com/dotnet/fsharp/issues/6818
- https://github.com/dotnet/fsharp/issues/9148
- https://github.com/dotnet/fsharp/issues/10289
- https://github.com/dotnet/fsharp/issues/11787
- https://github.com/dotnet/fsharp/issues/11969

**Comment by Happypig375** [comment]
https://github.com/dotnet/fsharp/issues/12824

**Comment by vzarytovskii** [comment]
> #12824

Yeah, there are a bunch of items missing, I'm no figuring out a way of automating such tracking issues based on tags
